app/javascript/types/ui.ts

```
@@ -1,0 +1,5 @@
+export interface ContainerProps {
+  children: React.ReactNode
```

ESLint

[error] 'React' is not defined.

(no-undef)